lib/omnibus/packagers/deb.rb

```diff
@@ -459,5 +459,23 @@ def safe_architecture
         Ohai['kernel']['machine']
       end
     end
+
+    #
+    # Install the specified packages
+    #
+    # @return [void]
+    #
+    def install(packages, _ = NULL)
+      `apt-get update && apt-get install -y --force-yes #{packages}`
```

Comment:
Using the `shellout` function provided by omnibus could be nicer to run commands (amongst other things, it logs the command and controls its output)